x/outline/access_key.go

```diff
@@ -25,15 +25,15 @@ import (
 	"github.com/Jigsaw-Code/outline-sdk/transport/shadowsocks"
 )
 
-type sessionConfig struct {
+type Prefix []byte
+
+type SessionConfig struct {
```

Comment:
This is not a good config format for reuse. It doesn't allow the composition our APIs allow. I'd rather keep that private for now.

For the OutlineDevice, you can provide a function that takes an access key and returns the transport objects.

Reply:
moved to `x/internal/outline`, that should be good for internal use only.